ctapipe/io/magiceventsource.py

```diff
@@ -0,0 +1,1337 @@
+        parsed_info = re.findall(mask, file_name)
+
+        run_number = int(parsed_info[0])
+
+        return run_number
+
+    @staticmethod
+    def load_events(file_list):
+        """
+        This method loads events from the pre-defiled file and returns them as a dictionary.
+
+        Parameters
+        ----------
+        file_name: str
+            Name of the MAGIC calibrated file to use.
+
+        Returns
+        -------
+        dict:
+            A dictionary with the even properties: charge / arrival time data, trigger, direction etc.
+        """
+
+        event_data = dict()
+
+        event_data['charge'] = []
+        event_data['arrival_time'] = []
+        event_data['trigger_pattern'] = np.array([])
+        event_data['stereo_event_number'] = np.array([])
+        event_data['pointing_zd'] = np.array([])
+        event_data['pointing_az'] = np.array([])
+        event_data['pointing_ra'] = np.array([])
+        event_data['pointing_dec'] = np.array([])
+        event_data['MJD'] = np.array([])
+
+        event_data['file_edges'] = [0]
+
+        for file_name in file_list:
+
+            input_file = uproot.open(file_name)
+
+            # Reading the info common to MC and real data
+            charge = input_file['Events']['MCerPhotEvt.fPixels.fPhot'].array()
+            arrival_time = input_file['Events']['MArrivalTime.fData'].array()
+            trigger_pattern = input_file['Events']['MTriggerPattern.fPrescaled'].array()
+            stereo_event_number = input_file['Events']['MRawEvtHeader.fStereoEvtNumber'].array()
+
+            # Computing the event arrival time
+            mjd = input_file['Events']['MTime.fMjd'].array()
+            millisec = input_file['Events']['MTime.fTime.fMilliSec'].array()
+            nanosec = input_file['Events']['MTime.fNanoSec'].array()
+
+            mjd = mjd + (millisec/1e3 + nanosec/1e9) / 86400.0
+
+            if 'MPointingPos.' in input_file['Events']:
+                # Retrieving the telescope pointing direction
+                pointing_zd = input_file['Events']['MPointingPos.fZd'].array() - input_file['Events']['MPointingPos.fDevZd'].array()
+                pointing_az = input_file['Events']['MPointingPos.fAz'].array() - input_file['Events']['MPointingPos.fDevAz'].array()
+                pointing_ra = (input_file['Events']['MPointingPos.fRa'].array() - input_file['Events']['MPointingPos.fDevHa'].array()) * 15.
+                pointing_dec = input_file['Events']['MPointingPos.fDec'].array() - input_file['Events']['MPointingPos.fDevDec'].array()
+            else:
+                # Getting the telescope drive info
+                drive_mjd = input_file['Drive']['MReportDrive.fMjd'].array()
+                drive_zd = input_file['Drive']['MReportDrive.fCurrentZd'].array()
+                drive_az = input_file['Drive']['MReportDrive.fCurrentAz'].array()
+                drive_ra = input_file['Drive']['MReportDrive.fRa'].array() * 15.
```

Comment:
If you could explain the magic number 15 here to a future reader of the code, I assume that would help readability a lot. 
One might put a comment or even nicer give a to this number, like:
```
number_of_foos_per_bar = 15
drive_ra = input_file['Drive']['MReportDrive.fRa'].array() * number_of_foos_per_bar
```
here the variable name `number_of_foos_per_bar` implies to the reader that `"MReportDrive.fRa"` is obviously measured in bars, while `drive_ra` is measured in foos and therefore the multiplication with a conversion constant was necessary. 

Anyway some kind of explanation of a magic number would be great I guess.

Reply:
Done.